### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.5)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59493]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
On the [/secure-messages/](https://staging.va.gov/my-health/secure-messages/) landing page, the card containing the "Go to your inbox" link has an H2 heading when you have unread messages ("45 unread messages in your inbox") and no H2 heading when you don't have any unread messages.

Headings should describe the content of the section they precede, and heading text and level should provide users information about the structure of the page. Screen reader users in particular depend on a well-defined heading structure to navigate through page content.

Here, the H2 heading is being used more as a status message. It's debatable whether it describes the content that follows it (just the "Go to your inbox" link), but if it does describe that content then there should always be a heading that describes that content. And since it appears/disappears conditionally, it may make it harder for screen reader users to build intuition about how the page is structured.

### Link, screenshot or steps to recreate
Steps to reproduce:
1. Log in using test user 41.
2. Note that on the Messages landing page, there is an H2 heading that indicates how many unread messages there are at the top of a gray card.
3. Log out.
4. Log in using test user 263.
5. Note that on the Messages landing page, the same gray card exists --- but with no heading.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

### Platform Recommendation
If it's important to have the number of unread messages in an H2 heading, then when there are no unread messages the card should have a heading that says "0 unread messages in your inbox" so that the structure of the page stays consistent.

But the current heading text seems more like a [status indicator(https://design.va.gov/components/card#status-via-label-with-indicator) than a heading. With that in mind, you could:
- Give the card a consistent heading (like "Your inbox") with a separate status indicator on the next line.
- Not have any heading on the card. The content here seems straightforward enough that it's probably not strictly necessary.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by lichellebain** [comment]
Jira ticket created:  https://jira.devops.va.gov/browse/MHV-46062
